Automattic/woocommerce-payments
# Increase GBP transaction limit for Afterpay

Fixes #7222 

#### Changes proposed in this Pull Request



- Afterpay/Clearpay BNPL txn upper limit in UK is 1200 GBP (instead of 1000 GBP)







#### Testing instructions




_You'll need a test store with a UK based merchant account, where GBP is available_

* As a shopper, select GBP in the multi-currency widget if you have other currencies available.
* Add product(s) to your order for a value greater than 1200 GBP
* Navigate to checkout
* Check that Afterpay is not an available payment method
* Remove product(s) from your order until the value is equal or less than 1200 GBP and greater than 1 GBP
* Navigate to checkout
* Select Afterpay from the available payment method
* Place the order, you should be redirected to Stripe test page.
* In Stripe test page, Authorize payment and you should be redirected to the Order Received page


-------------------


**Post merge**
